react/components/LogoutCustomerSession.tsx

```diff
@@ -49,6 +53,24 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm();
+
+  const id = orderForm.id;
+
+  const [customerClass, setCustomerClass] = useState("");
+
+  function getCustomerClass() {
+    var requestOptions = {
+      method: 'GET'
+    }
+  
+    return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+
+  }
+
+  getCustomerClass().then(response => response.json()).then(data => {
+    setCustomerClass(data.clientProfileData.customerClass);
+  })
```

Comment:
This causes an infinite loop. Can you wrap it with an useEffect?!

Reply:
@igorbrasileiro Updated